src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracMonitor.java

```diff
@@ -117,7 +119,7 @@ public void run() {
                             DiracStatus status = DiracStatus.valueOf(res[1].replace("Status=", "").replace(";", ""));
                             String diracSite = siteRes[2].replace("Site=", "").replace(";", "").trim();
 
-                            if ( (!diracSite.equalsIgnoreCase("ANY")) && (job.getDiracSite()==null) ) {
+                            if ((!diracSite.equalsIgnoreCase("ANY")) && (job.getDiracSite() == null)) {
```

Comment:
I liked the spaces between the parenthesis to make the condition clearer. Do you prefer it that way ?

Reply:
No, not really, I'll put them back :-)